src/server/identity/server/api_server_test.go

```diff
@@ -274,3 +275,39 @@ func TestIDPConnectorCRUD(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 0, len(listResp.Connectors))
 }
+
+// TestShortenIDTokenExpiry tests that we can configure Dex to issue ID tokens with a
+// expiration shorter than the default of 6 hours
+func TestShortenIDTokenExpiry(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	tu.DeleteAll(t)
+	defer tu.DeleteAll(t)
+
+	tu.ConfigureOIDCProvider(t)
+
+	adminClient := tu.GetAuthenticatedPachClient(t, auth.RootUser)
+	_, err := adminClient.SetIdentityServerConfig(adminClient.Ctx(), &identity.SetIdentityServerConfigRequest{
+		Config: &identity.IdentityServerConfig{
+			Issuer:        "http://localhost:30658/",
+			IdTokenExpiry: "1h",
```

Comment:
maybe this is a trivial question, but is it worth calling out somewhere the format of the Expiry string?